src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+    Order add(Order order);
+
+    List<Order> getAll(User user);

```

Comment:
readme: - OrderDao. To get the order for a specific user create method getByUser().

The same for OrderDaoImpl, OrderServiceImpl